tests/integration/devfile/cmd_devfile_catalog_test.go

```diff
@@ -77,4 +77,35 @@ var _ = Describe("odo devfile catalog command tests", func() {
 			helper.MatchAllInOutput(output, wantOutput)
 		})
 	})
+
+	Context("When executing catalog describe component with a component name with a single component", func() {
+		It("should only give information about one component", func() {
+			output := helper.CmdShouldPass("odo", "catalog", "describe", "component", "maven")
+			helper.MatchAllInOutput(output, []string{"generateName: maven-", "- command: java -jar target/*.jar"})
+		})
+	})
+	Context("When executing catalog describe component with a component name with multiple components", func() {
+		It("should print multiple devfiles from different registries", func() {
+			output := helper.CmdShouldPass("odo", "catalog", "describe", "component", "nodejs")
```

Comment:
I am confused with the term `multiple components` that you used in Context. Actually I am not sure how it describe the multiple component with `odo catalog describe component nodejs`.

Reply:
Ahhh... I understood now Thanks